torchci/rockset/commons/filter_forced_merge_pr.lambda.json

```diff
@@ -8,5 +8,4 @@
     }
   ],
   "description": "Check if these PRs are forced merge"
-}
-
+}
```

Comment:
Is the removal of the new line intentional?

Reply:
this was generated by running `yarn node scripts/downloadQueryLambda.mjs commons filter_forced_merge_pr`, i think some of the other .lambda.json files are like this too so i assume its fine